src/applications/facility-locator/components/search-results/LocationInfoBlock.jsx

```diff
@@ -16,7 +16,7 @@ const LocationInfoBlock = ({ location }) => {
       {isProvider ? (
         <span>
           <Link to={`provider/${location.id}`}>
-            <h5>{name}</h5>
+            <h2 className="vads-u-font-size--h5">{name}</h2>
```

Comment:
Do you know where I can find a list of all of these class names? Do we have that in the style guide perhaps?

Reply:
The design system! This one doesn't fit the typical model of utility but it's such a popular use case to make one type of heading match the size of another that it was added, https://design.va.gov/utilities/font-size.

Also, check out the `Responsive prefix` section, where you can control how an element renders on different screen sizes via a classname.

Reply:
Ah, I think this may have all of them 👍 https://design.va.gov/utilities/

Reply:
Oops posted my previous comment before GitHub could update with your comment haha Thanks!